qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -750,7 +750,24 @@ def main(args=None):
         kde=args.kde)
 
     if args.watch:
-        with daemon.pidfile.TimeoutPIDLockFile(args.pidfile):
+        fd = os.open(args.pidfile,
+                     os.O_WRONLY | os.O_CREAT | os.O_CLOEXEC,
+                     0o600)
+        with os.fdopen(fd, 'rw') as lock_f:
+            try:
+                fcntl.flock(fd, fcntl.LOCK_EX | fcntl.LOCK_NB)
+            except BlockingIOError:
+                try:
+                    pid = int(lock_f.read().strip())
+                except Exception:
```

Comment:
`ValueError`? IMO better not silently ignore `IOError` here, as it may suggest something really wrong happening.

Reply:
Ah, good point.